api/v1/stats/issues/StatsIssueHandler.php

```diff
@@ -294,6 +295,7 @@ public function getTimeline(SlimHttpRequest $slimRequest, APIResponse $response,
             'dateStart',
             'dateEnd',
             'timelineInterval',
+            'type'
```

Comment:
Do we have an issue filed somewhere to document the new stats api endpoints?

Reply:
yes, the whole documentation issue is here: https://github.com/pkp/pkp-lib/issues/8014
(and https://github.com/pkp/pkp-lib/issues/7953)